CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+
+1.5.1 / 2023-05-10
+==================
+This release contains an important Native targets overhaul, as well as numerous enhancements and bugfixes.
+Kotlin 1.8.21 is used by default.
+
+### New set of Native targets
+
+Kotlin/Native team recently has published a [document](https://kotlinlang.org/docs/native-target-support.html) describing 
```

Comment:
It's not "Kotlin/Native team" but rather "The official Kotlin target's support policy"